trezor_agent/client.py

```diff
@@ -28,6 +28,19 @@ def get_public_key(self, identity):
         return formats.export_public_key(vk=vk,
                                          label=str(identity))
 
+    def get_public_keys(self, identities):
```

Comment:
@romanz any particular reason why this was dropped? The current implementation will enumerate devices, acquire one, retrieve the public key and release the device for every defined identity. This is fine if you only have a few identities, but it can become slow if there are many. With this implementation we only enumerate and acquire once, retrieve all public keys and then release.
Does it make sense?